samples/cellular/nrf_cloud_multi_service/overlay_nrf7002ek_wifi_no_lte.conf

```diff
@@ -61,7 +61,9 @@ CONFIG_WIFI_MGMT_EXT=y
 CONFIG_WIFI_CREDENTIALS_BACKEND_SETTINGS=y
 
 ## Use compile-time client ID for nRF Cloud
-CONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME=y
+CONFIG_NRF_CLOUD_CLIENT_ID_SRC_HW_ID=y
+CONFIG_HW_ID_LIBRARY=y
+CONFIG_HW_ID_LIBRARY_SOURCE_NET_MAC=y
```

Comment:
Similarly, you could enable this in your test, rather than in the overlay. I would rather not prescribe a specific means of getting ID in this overlay, it's meant to be compatible with whatever the user configures themselves

Reply:
i didn't mean to approve.  i think this should be removed?